.github/PR_TEMPLATE/pull_request.md

```diff
@@ -0,0 +1,55 @@
+---
+name: Pull Request
+about: Increment new features to kool.dev
+title: ""
+labels: ""
+assignees: ""
+---
+
+## **Acknowledgements**
+
+:speech_balloon: The code I am submitting is commented.
+
+:white_check_mark: The code I am submitting is Tested, and I will describe if there are any side effects **(including potential)**.
+
+:relieved: I understand that my request may be rejected, and will **not** take this personally.
+
+:cool: If I found an issue, I have already open a issue or alerted the maintainers, heterofore submitting this PR for review.
+
+## **TL;DR**
+
+Provide a short description on what you are implementing. Refference errors, or anything that makes it very clear after a single read what is being changed. Focus on write less, while making yourself clear.
+
+<hr>
+
+**Describe the update**
+
+A clear explanation of what is being updated. Try to remember these points:
+
+-   Why is this Important?
+-   Are there side effects ? (if so list them)
+-   Is there something relevant to a specific commit ?
+-   Are you solving a issue? if so don't forget to note which and explain your solution process
+
+**Kool version and environment**
+
+Please provide the version you are running (`kool -v`), which environment (OS, shell, docker, DinD), and also what environment variables you are using (`kool info`).
+
+E.g:.
+
+```bash
+~$ kool -v
+kool version 1.7.0
+```
+
+**Expected behavior**
+
+A clear and concise description of what you expected to happen, followed by a description on implemented tests.
```

Comment:
I guess we can simplify the message of this topic so simply: `How to test/verify the change`.